src/Status.bonus.test.js

```diff
@@ -0,0 +1,23 @@
+import { mount } from "enzyme";
+
+import { Status } from "./Status";
```

Comment:
expect `Status` to be a default export (in keeping with what the fellows are being taught for now)

Reply:
To keep this bonus spec from running in Circle, you'll need to add a `testPathIgnorePatterns` flag to `package.json`. Then a separate `test:bonus` script to run all tests including the bonus. Example: https://github.com/joinpursuit/Pursuit-Core-Web-Props-Lab-Tested/blob/main/package.json#L18-L19